std/array.d

```diff
@@ -414,14 +414,17 @@ Construct a range iterating over an associative array by key/value tuples.
 Params: aa = The associative array to iterate over.
 
 Returns: A $(REF_ALTTEXT forward range, isForwardRange, std,_range,primitives)
-of Tuple's of key and value pairs from the given associative array.
+of Tuple's of key and value pairs from the given associative array. The members
+of each pair can be accessed by name (`.key` and `.value`). or by integer
+index (0 and 1 respectively).
 */
 auto byPair(Key, Value)(Value[Key] aa)
 {
-    import std.typecons : tuple;
+    import std.typecons : Tuple;
     import std.algorithm.iteration : map;
 
-    return aa.byKeyValue.map!(pair => tuple(pair.key, pair.value));
+    alias NamedTuple = Tuple!(Key, "key", Value, "value");
+    return aa.byKeyValue.map!(pair => NamedTuple(pair.key, pair.value));
```

Comment:
Thanks, I always forget that the free function takes the names separately (which is funny because I worked a lot on that stuff getting `Tuple.rename` in)